pkgs/tools/text/grin/default.nix

```diff
@@ -1,16 +1,16 @@
-{ stdenv, fetchurl, python2Packages }:
+{ stdenv, fetchgit, python2Packages }:
 
 python2Packages.buildPythonApplication rec {
   name = "grin-1.2.1";
   namePrefix = "";
 
-  src = fetchurl {
-    url = "mirror://pypi/g/grin/${name}.tar.gz";
-    sha256 = "1swzwb17wibam8jszdv98h557hlx44pg6psv6rjz7i33qlxk0fdz";
+  src = fetchgit {
```

Comment:
In general I'd recommend `fetchFromGitHub` when using `fetchgit` against github.com, since it's quite a bit more efficient.